manifests/p/PuTTY-CAC/PuTTY-CAC/0.76u4/PuTTY-CAC.PuTTY-CAC.installer.yaml

```
@@ -0,0 +1,34 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: PuTTY-CAC.PuTTY-CAC
+PackageVersion: 0.76u4

```

```suggestion
PackageVersion: 0.76.0.4
```
This must match Control Panel.